### PR TITLE
Expose samples for SCAPI

### PR DESCRIPTION
**Pull request by aure-olivier**
Adding :

- LICENSE (MIT)
- sample.html (cleaned and reformatted)
- wwe-service-client-api.js (reformatted)
- Some content in README.md in order to navigate to SCAPI samples

### CODE REVIEW

**Comment by sblecon** [comment]
Fine with the proposal

**Review by tichon29 — APPROVED** [review]

File: wwe-service-client-api/sample.html [hunk 1]
```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
```
Comment (sblecon) [inline wwe-service-client-api/sample.html]: I suggest to rename the file by index.html?

**Review by cadgerfeast — APPROVED** [review]

File: wwe-service-client-api/sample.html (hunk 1 above)
Comment (aure-olivier) [reply wwe-service-client-api/sample.html]: Sure, will rename that file.

File: wwe-service-client-api/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+service-client-api.accepted-web-content-origins=http://my-web-server
+
+[MyCustomAPISample]
+label=My Sample
+url=http://my-web-server/path/sample.html
+```
+
+Click on the following assets :
+
+- [sample.html](./sample.html) (custom webpage)
```
Comment (sblecon) [inline wwe-service-client-api/README.md]: should be renamed here also for sample.html

Comment (aure-olivier) [reply wwe-service-client-api/README.md]: Yes, just pushed it ^^

File: wwe-service-client-api/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+service-client-api.accepted-web-content-origins=http://my-web-server
+
+[MyCustomAPISample]
+label=My Sample
+url=http://my-web-server/path/sample.html
+```
+
+Click on the following assets :
+
+- [sample.html](./index.html) (custom webpage)
```
Comment (sblecon) [inline wwe-service-client-api/README.md]: I still sample.html in this file

Comment (aure-olivier) [reply wwe-service-client-api/README.md]: Ok, my bad I forgot to update link name :)

**Review by sblecon — APPROVED** [review]

**Pull request merged**